### PR TITLE
Migrate iOS to v5

### PR DESCRIPTION
**Pull request by descorp**
# Open PR

## Changes

* migrate to v5
* rename `XxxComponent` into `XxxModule`

### CODE REVIEW

File: example/ios/AdyenExampleTests/ApplePayConfigurationTests.swift [hunk 1]
```diff
@@ -6,13 +6,14 @@
...
 import Adyen
-import adyen_react_native
+@testable import adyen_react_native
 import PassKit
 
 final class ApplePayConfigurationTests: XCTestCase {
   
   let mockAmount = Amount(value: 1000, currencyCode: "USD", localeIdentifier: "en-US")
-  
+  lazy var  mockPayment = Payment(amount: mockAmount, countryCode: "US")
```
Comment (goergisn) [inline example/ios/AdyenExampleTests/ApplePayConfigurationTests.swift]:
```suggestion
  lazy var mockPayment = Payment(amount: mockAmount, countryCode: "US")
```

File: ios/Components/AdyenApperance.swift [hunk 2]
```diff
@@ -4,9 +4,10 @@
 // This file is open source and available under the MIT license. See the LICENSE file for more info.
 //
 
-import Adyen
+@_spi(AdyenInternal) import Adyen
```
Comment (goergisn) [inline ios/Components/AdyenApperance.swift:7]: Why is it suddenly needed?

File: ios/Components/DropInModule.swift [hunk 3]
```diff
@@ -35,22 +35,29 @@ internal final class AdyenDropIn: BaseModule {
             return sendEvent(error: error)
         }
 
-        let apiContext = APIContext(environment: parser.environment, clientKey: clientKey)
+        AdyenLogging.isEnabled = true
```
Comment (goergisn) [inline ios/Components/DropInModule.swift]: Not sure if this belongs here?

Comment (descorp) [reply ios/Components/DropInModule.swift]: 👍

File: ios/Components/AdyenApperance.swift (hunk 2 above)
Comment (descorp) [reply ios/Components/AdyenApperance.swift:7]: for `adyenPrint`
I should probably come up with better way of logging things

File: example/ios/AdyenExampleTests/ApplePayConfigurationTests.swift (hunk 1 above)
Comment (descorp) [reply example/ios/AdyenExampleTests/ApplePayConfigurationTests.swift]: 👍

**Review by goergisn — DISMISSED** [review]

**Review by goergisn — APPROVED** [review]

**Pull request merged**